api/GlobalFetch.json

```diff
@@ -0,0 +1,117 @@
+{
+  "api": {
+    "GlobalFetch": {
+      "__compat": {
+        "mdn_url": "https://developer.mozilla.org/docs/Web/API/GlobalFetch",
+        "support": {
+          "webview_android": {
+            "version_added": "42"
+          },
+          "chrome": {
+            "version_added": "42"
+          },
+          "chrome_android": {
+            "version_added": "42"
+          },
+          "edge": {
+            "version_added": true
+          },
+          "edge_mobile": {
+            "version_added": true
+          },
+          "firefox": [
+            {
+              "version_added": "39"
+            },
+            {
+              "version_added": "41",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
+                }
+              ]
+            }
+          ],
+          "firefox_android": {
+            "version_added": false
+          },
+          "ie": {
+            "version_added": false
+          },
+          "opera": [
+            {
+              "version_added": "29"
+            },
+            {
+              "version_added": "28",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": ""
```

Comment:
Both versions of Opera should match meaning both should say added in 29. I think MDN folks would like you to keep the flag information (that it was behind a flag in 28.) That should also be the same for both versions of Opera.

Reply:
👍